client/src/components/material.jsx

```diff
@@ -59,24 +43,102 @@ const MatDesc = Styled(MatName)`
 const Program = Styled(SmallLeft)`
   grid-area: LeftButton;
   grid-column-start: 1;
+  margin: 0px 25px 45px 25px;
+  padding: 5px 20px;
 `
 const LearnMore = Styled(SmallRight)`
   grid-area = RightButton;
   grid-column-start: 2;
   grid-row-start: 4;
+  margin: 0px 25px 45px 25px;
+  padding: 5px 20px;
+`
+const MatInfo = Styled.div`
+  font-family: 'Nunito Sans', sans-serif;
+  font-size: 0.82rem;
+  color: #fff;
+  transition: .5s ease;
+  opacity: 0;
+  text-align: center;
+`
+
+const Card = Styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1.6fr 0.4fr 1.25fr 0.75fr;
+  gap: 0px 0px;
+  overflow: hidden;
+  grid-template-areas:
+    ". blob"
+    "MatName MatName"
+    "MatDesc MatDesc"
+    "LeftButton RightButton";
+  background-color: #f0f4f5;
+  font-family: 'Nunito Sans', sans-serif;
+  cursor: pointer;
+  border-radius: 5%;
+  margin: 10px;
+  width: 320px;
+  height: 450px;
+  &&:hover ${LearnMore} {
+    background-color: #000;
+    color: #fff;
+    z-index: 10;
+  }
+  &&:hover ${MatInfo} {
+    opacity: 1;
+    z-index: 10;
+  }
+
+  &&:hover {
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    background: url("https://www.patagonia.com/dw/image/v2/bdjb_PRD/on/demandware.static/-/Library-Sites-PatagoniaShared/default/dw6fbb2428/fpc/programs/brewer_c_0001_cc_WEB-1200x1200.jpg?q=60&amp;sw=600&amp");
+    z-index: 10;
+    opacity: .8;
+    ${MatDesc} {
+      opacity: 0;
+    };
+    ${MatName} {
+      opacity: 0;
+    };
+    ${Blob} {
+      opacity: 0;
+    };
+    ${Program} {
+      background: transparent;
+      color: #fff;
+    }
+  }
 `
 
 class Material extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      isHovered: false
+    }
+
+    this.handleHover = this.handleHover.bind(this);
+  }
+
+
+  handleHover(event) {
+    event.preventDefault();
+    console.log(event.target.id)
+    this.setState(prevState => {
+      return {
+        isHovered: !prevState.isHovered
+      }
+    })
   }
 
   render() {
     const materials = this.props.materials;
-  
     let matDisplay = materials.map(material => {
       const { id, mat_desc, mat_info, mat_name } = material;
-      return <Card key={id}>
+      return <Card key={id} onMouseEnter={this.handleHover} onMouseLeave={this.handleHover}>
+        {this.state.isHovered ? <MatInfo>{mat_info.slice(0, 250)}</MatInfo> : null }
```

Comment:
This is awesome! I have not seen onMouseEnter and onMouseLeave before but what you did here is so cool and I learned from this!

I just went through the react docs and found these in there and several others. So cool.